06week/checkpoint2.js

```diff
@@ -0,0 +1,126 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+
+const nums = strNums.map((number) => {
+  return Number(number)
+});
+
+console.log(nums);
+
+// Find the sum of the even values
+const sumEvens = nums.filter((number) => {
+
```

Comment:
only line break once within a function to improve readability.  Also, don't use reserved JS words like number when declaring variables

This is a bit easier to read:
```
const sumEvens = nums.filter(num => {
  return num % 2===0
}).reduce((accumulator, currentValue) => { 
  return accumulator + currentValue;  
});
```